.github/workflows/api-tests.yml

```diff
@@ -95,8 +95,8 @@ jobs:
               environment:
                 - PMM_DEBUG=1
                 - PERCONA_TEST_CHECKS_INTERVAL=10s
-                - PERCONA_TEST_PLATFORM_ADDRESS=https://check-dev.percona.com
-                - PERCONA_TEST_PLATFORM_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
+                - PMM_PERCONA_TEST_PLATFORM_ADDRESS=https://check-dev.percona.com
+                - PMM_PERCONA_TEST_PLATFORM_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
```

Comment:
let's make it
`PMM_TEST_PERCONA_PLATFORM_ADDRESS` and `PMM_TEST_PERCONAT_PLATFORM_PUBLIC_KEY`